app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'comment'
```

Comment:
- [ ] Same here the `post` has many `comments` and `likes`, and it belongs_to `author`, kindly make sure to set up these associations so your modules work properly